src/app/users/user-add/user-add-form-data.ts

```diff
@@ -161,6 +161,15 @@ export const FORM_DATA_USER_ADD = [
         newLine: true,
         className: 'size-large',
       },
+      {
+        ...FORM_DATA_INPUT,
+        key: 'healthFacilityDetails',
+        label: 'captions.CaseData.healthFacilityDetails',
+        newLine: true,
+        className: 'size-full',
+        dependingOn: 'address__Facility',
```

Comment:
this should match the key (as in address.facility not with __)
also, keys match the attributes in dtos, so they can't start with uppercase; please correct to have address.facility